docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -98,6 +98,10 @@ The following configuration properties are available:
    * - ``http-server.authentication.oauth2.user-mapping.file``
      - File containing rules for mapping user. See :doc:`/security/user-mapping`
        for more information.
+   * - ``http-server.authentication.oauth2.principal-field``
+     - The field of the access token used for the Trino user principal.
+       Example: ``sAMAccountName,name,sub,upn``
```

Comment:
I'm not sure I understand this example.
Please mention that `sub` is the default.

Reply:
@mosabua @lukasz-walkiewicz 
What I wanted to convey here is that it can be any other field from your decoded JWT token. As a user, I was also not aware of this so thought of mentioning other possible fields.  I agree this is a bit confusing. How about this

`The field of the access token used for the Trino user principal. The default value is sub but other fields can also be configured`
Possible values: `sAMAccountName`,`name`,`upn`,`email` etc
?
any other suggestions?

Reply:
How about

```
The field of the access token used for the Trino user principal. Defaults to ``sub``. Other commonly used fields include ``sAMAccountName``, ``name``, ``upn``, and ``email``.
```